src/_community/rules.md

```diff
@@ -46,14 +46,26 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted **in all Quilt community spaces** must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, mods
+  that modify Quilt in an attempt to hide themselves from servers they connect to, or mods that harass specific people 
```

Comment:
Can see a problem here - what about mods that kick people without a certain permission at spefic times?

Reply:
I wouldn't consider that harassment - this rule is about mods that, for example, hardcode UUIDs to kick from all servers for a grudge, or that provide a back-door that allows the developer to shut down a server.

The example given is here for brevity - security mods are fine.